ntoskrnl/se/srm.c

```diff
@@ -253,7 +254,7 @@ SeRmInitPhase1(VOID)
                                   NULL);
     if (!NT_SUCCESS(Status))
     {
-        DPRINT1("Security: Rm Server Thread creation failed 0x%lx\n", Status);
+        DPRINT1("Security: Rm Command Server Thread creation failed: 0x%lx\n", Status);
```

Comment:
```suggestion
        DPRINT1("Security: Rm Create Server Thread failed 0x%lx\n", Status);
```

Reply:
This file does not look fully consistent about its error message formats...
My point, for these 3 occurrences, is to better match their related variable names and add a colon, while making them consistent within this function.

If you want to reduce wording and other reviewers agree, I would suggest a simple pattern like
`PsCreateSystemThread(SepRmCommandServerThread) failed: 0x%lx\n`.
Meanwhile, I would like that not to block this PR further.